components/ILIAS/Contact/classes/class.ilMailSearchObjectGUI.php

```diff
@@ -313,20 +229,12 @@ protected function mailObjects(): void
     public function mailMembers(): void
     {
         $members = [];
-        $usr_ids = [];
-        if ($this->http->wrapper()->query()->has('search_members')) {
-            $usr_ids = [
-                $this->http->wrapper()->query()->retrieve(
-                    'search_members',
-                    $this->refinery->kindlyTo()->int()
-                )
-            ];
-        } elseif ($this->http->wrapper()->post()->has('search_members')) {
-            $usr_ids = $this->http->wrapper()->post()->retrieve(
-                'search_members',
-                $this->refinery->kindlyTo()->listOf($this->refinery->kindlyTo()->int())
-            );
-        }
+
+        $usr_ids = $this->http->wrapper()->query()->has('contact_mailinglist_search_members_ids')
```

Comment:
This is not a `dictOf`, but a `listOf`. Please also try to use `byTring` and `always` to avoid the `has` check.